sympy/functions/elementary/complexes.py

```diff
@@ -911,9 +909,29 @@ def _eval_evalf(self, prec):
         return (ub - ceiling(ub/period - S(1)/2)*period)._eval_evalf(prec)
 
 
-def unbranched_argument(arg):
-    from sympy import oo
-    return periodic_argument(arg, oo)
+def unbranched_argument(ar):
+    from sympy import exp_polar, log, polar_lift, oo
```

Comment:
The imports should be at the top of the file, if there are no import cycles. `exp_polar` is there already. `polar_lift` is in this file.